support/k8s/k8s-workload-registrar/mode-crd/webhook/webhook_svid.go

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
We haven't traditionally added the copyright clause to each source file and for the sake of consistency should probably exclude it here. Whether or not adding the clause is a good idea is topic of debate 😆